patches/lzma-native-index-static-addon-require.patch

```diff
@@ -1,17 +1,15 @@
-diff --git a/index.js b/index.js
-index 1e7c766..e42c513 100644
---- a/index.js
-+++ b/index.js
+--- b/node_modules/lzma-native/index.js	2018-01-23 14:37:50.000000000 -0400
++++ a/node_modules/lzma-native/index.js	2018-01-23 14:37:00.000000000 -0400
 @@ -7,11 +7,8 @@ var extend = require('util-extend');
  var assert = require('assert');
  var fs = require('fs');
-
+ 
```

Comment:
What's with the whitespace here? Is that a space or just a newline?

Reply:
They came from `lzma-native`. I was able to make the patch apply without them by using `--ignore-whitespace`